src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -74,6 +78,18 @@ protected function createFactory()
     }
 
     /**
+     * Create an API resource for the model.
+     *
+     * @return void
+     */
+    protected function createApiResource()
+    {
+        $this->call('make:resource', [
```

Comment:
Isn't this creating a normal resource controller instead of the API one?

Reply:
To create a resource controller, it's `php artisan make:controller UsersController -r`, whereas `php artisan make:resource User` is for an API resource. As stated above, it's going to get easy confused.
We now have `resources` for assets, views, and lang, and `app/Http/resources` for the API resources. I'd suggest renaming this as happened with commands and jobs. (https://twitter.com/taylorotwell/status/569941495127547904?lang=en)